scripts/zones/The_Ashu_Talif/instances/the_black_coffin.lua

```diff
@@ -6,6 +6,28 @@ local ID = zones[xi.zone.THE_ASHU_TALIF]
 -----------------------------------
 local instanceObject = {}
 
+local crewTable =
+{
+    -- The Black Coffin - Wave 1
+    [1] =
+    {
+        ID.mob.ASHU_CREW_OFFSET,
+        ID.mob.ASHU_CREW_OFFSET + 1,
+        ID.mob.ASHU_CREW_OFFSET + 2,
+        ID.mob.ASHU_CREW_OFFSET + 3,
+        ID.mob.ASHU_CREW_OFFSET + 4,
+    },
+    -- The Black Coffin - Wave 2
+    [2] =
```

Comment:
```lua
ASHU_CREW_OFFSET    = GetFirstID('Ashu_Talif_Crew'), <- This would be [1]
ASHU_CAPTAIN_OFFSET = GetFirstID('Ashu_Talif_Captain'), <- This would be [2]
```
I think

Reply:
I can add comments in the ID lua if you want to make it more clear. I thought if I did +5 +6 +7 +8 etc. it would of been more confusing.